engine/api/src/main/scala/pl/touk/nussknacker/engine/api/definition/NodeDependency.scala

```diff
@@ -44,14 +44,14 @@ object Parameter {
   // you should redefine scalaOptionParameter and javaOptionalParameter
   def optional(name: String, typ: TypingResult): Parameter =
     Parameter(name, typ, editor = None, validators = List.empty, additionalVariables = Map.empty,
-      branchParam = false, isLazyParameter = false, scalaOptionParameter = false, javaOptionalParameter = false)
+      branchParam = false, isLazyParameter = false, isRequiredParameter = false, scalaOptionParameter = false, javaOptionalParameter = false)
 
 }
 
 object NotBlankParameter {
 
-  def apply(name: String, typ: TypingResult): Parameter =
-    Parameter(name, typ, validators = List(NotBlankParameterValidator))
+  def apply(name: String, typ: TypingResult, isRequiredParameter: Boolean = true): Parameter =
```

Comment:
`NotBlankParametere` sould be always required?  I mean there shouldn't be possiblity to set this.